sync/class.jetpack-sync-module-updates.php

```diff
@@ -28,6 +31,12 @@ public function init_listeners( $callable ) {
 		), 10, 2 );
 
 		add_action( 'automatic_updates_complete', $callable );
+
+		// Send data when update completes
+		add_action( '_core_updated_successfully', array( $this, 'update_core' ) );
+		add_action( 'jetpack_sync_update_core_successfull', $callable, 10, 2 );
+		add_action( 'jetpack_sync_autoupdate_core_successfull', $callable, 10, 2 );
+		add_action( 'jetpack_sync_reinstall_core_successfull', $callable, 10, 1 );
```

Comment:
When you have priority 10 and a single argument, can't you omit these parameters? This isn't a blocking change, of course.